base/inc/fields/siteorigin-widget-field-class-loader.class.php

```diff
@@ -29,29 +30,35 @@ static function single(){
 	/**
 	 * Regsiter class prefixes to watch for in this loader
 	 *
-	 * @param $class_prefixes
+	 * @param string|array $class_prefixes
+	 * @param string $group
 	 */
-	public function add_class_prefixes( $class_prefixes ) {
+	public function add_class_prefixes( $class_prefixes, $group = 'core' ) {
```

Comment:
The main reason I'd want this default is for backwards compatibility. I don't know of any plugins that are using this function, but it's possible some devs have. Either way, this is in the spirit of backwards compatibility - if there is such a thing.